compactHeadersApi.js

```diff
@@ -190,6 +190,67 @@ function install(window) {
     msgHeaderView.append(compactHeadersPopup);
   }
 
+  function patchRecipientClass() {
+    window.customElements.whenDefined("header-recipient").then(classHeaderRecipient => {
+      if (!classHeaderRecipient.prototype.originalUpdateRecipient) {
+        classHeaderRecipient.prototype.originalUpdateRecipient = classHeaderRecipient.prototype.updateRecipient;
+        classHeaderRecipient.prototype.updateRecipient = function() {
+          this.originalUpdateRecipient();
+
+          if (this.dataset.headerName == "to" || this.dataset.headerName == "cc") {
+            this.nameLine.textContent = this.displayName;
+            this.addressLine.textContent = this.emailAddress;
+            if (this.displayName) {
+              this.classList.add("has-display-name");
+            } else {
+              this.classList.remove("has-display-name");
+            }
+          }
+        }
+        // Call updateRecipient for existing recipients before patched
+        for (let recipient of expandedtoBox.querySelectorAll('li[is="header-recipient"]')) {
+          recipient.updateRecipient();
+        }
+        for (let recipient of expandedccBox.querySelectorAll('li[is="header-recipient"]')) {
+          recipient.updateRecipient();
+        }
+      }
```

Comment:
We call `updateRecipient` for existing recipients during patching, because `install` is called in `onMessageDisplayed` event, which is triggered after the message page is ready.